src/components/ec-file-list/ec-file-list.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    data-test="ec-file-list"
+    class="ec-file-list"
+  >
+    <div
+      v-for="(item, index) in items"
+      :key="item.name"
```

Comment:
If we use as key the name instead of the index we have the risk of triggering a Vue warning in case two files have the same name